packages/@aws-cdk/aws-stepfunctions/test/state-machine-resources.test.ts

```diff
@@ -587,6 +588,31 @@ describe('State Machine Resources', () => {
     });
   }),
 
+  test('Imported state machine can provide metrics', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const stateMachineArn = 'arn:aws:states:::my-state-machine';
```

Comment:
```suggestion
    const stateMachineArn = 'arn:aws:states:us-east-1:123456789012:stateMachine:my-state-machine';
```

rationale: use a state machine arn that's representative of a real arn rather than an arbitrary arn like string